radar_mobileye_data[] objs
